release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -539,6 +539,13 @@ Benefits of this feature include faster data processing, reduced CPU workloads,
 
 For more information, see xref:../networking/hardware_networks/configuring-hardware-offloading.adoc#about-hardware-offloading_hardware-offloading[Configuring hardware offloading].
 
+[id="ocp-4-10-networking-external-dns-operator"]
+==== Creating DNS records by using Red Hat External DNS Operator (Technology Preview)
+
+You can now create DNS records by using Red Hat External DNS Operator on cloud providers such as AWS, Azure and GCP. You can install the External DNS Operator using the Red Hat OpenShift Container Platform Operator Hub. You can use configurational parameters to configure `ExternalDNS` as per your need. 
```

Comment:
A few minor suggestions, just remember not to click the "commit suggestion" button, and instead make the updates manually.

```suggestion
You can now create DNS records by using the Red Hat External DNS Operator on cloud providers such as AWS, Azure, and GCP. You can install the External DNS Operator using OperatorHub. You can use parameters to configure `ExternalDNS` as required. 
```

Reply:
Addressed